qcodes/logger/logger.py

```diff
@@ -332,16 +333,19 @@ def log_qcodes_versions(logger: logging.Logger) -> None:
     """
     Log the version information relevant to QCoDeS. This function logs
     the currently installed qcodes version, whether QCoDeS is installed in
-    editable mode, and the installed versions of QCoDeS' requirements.
+    editable mode, the installed versions of QCoDeS' requirements, and the
+    versions of all installed packages.
```

Comment:
Do we want to keep logging both qcodes requirements and all installed packages. I guess we can add this now and eventually remove the other?

Reply:
Yes, that sounds like a good approach. The requirements call adds no new information, although it sometimes conveniently marks a package "not installed". Let's remove it after the installed packages have been logged in one or two releases.